app/page.tsx

```diff
@@ -1,3 +1,46 @@
+import Link from "next/link";
+
 export default function HomePage() {
-  return <h1>swig.dev</h1>;
+  return (
+    <div className="space-y-8">
+      <div>
+        <input
+          type="text"
+          placeholder="search events..."
+          className="w-full border-b border-gray-200 bg-transparent py-2 focus:border-orange-600 focus:outline-none"
+        />
+      </div>
+      <div>
+        <h2 className="mb-4">nearby events</h2>
```

Comment:
margin is used a lot where I think padding or space-* would've been the correct choice